<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarCluster</class>
 <widget class="QMainWindow" name="CarCluster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CarCluster</string>
  </property>
  <widget class="QWidget" name="clusterWindowWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000080;</string>
   </property>
   <widget class="QFrame" name="sliderFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>339</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="speedMeterWidget" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
   <widget class="QWidget" name="rpmMeterWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="systemInfoLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244)</string>
    </property>
    <property name="text">
     <string>Current Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
